src/Pages/Products/index.js

```diff
@@ -0,0 +1,86 @@
+// import { Link } from 'react-router-dom';
+
+import './index.css';
+import '../../styles/base.css';
+import { ProductCard } from '../../components/Card';
+
+const ProductFilterComp = ({ children, className }) => {
+  return <div className={className}>{children}</div>;
+};
+
+const ProductFilter = ({ label, type }) => {
+  return (
+    <div className='prod-func'>
+      <input type={type} />
+      <label>{label}</label>
+    </div>
+  );
+};
+
```

Comment:
suggestion: move these components to a different file

Reply:
These files will not be used anywhere, that's why they are here